### ISSUE TITLE
Crosshair when taking region-capture screenshots

### ISSUE THREAD

**Issue by BloodRaven0** [ShareX/ShareX#6273]
The only feature I miss from switching over from FreeShooter is the horizontal and vertical lines starting from your cursor when taking region-capture screenshots.
The lines are basically guides as to where two of your borders are going to be, which I think is important.
They don't have to be bright and obnoxious like FreeShooter's crosshairs - they could just use the normal (source) image brightness (when taking region capture, the entire source is darkened), or slightly brighten that in order to have some contrast. It could be 1 pixel wide (or 3, with the middle line being brightest for high-dpi displays like 4k).

**Comment by Jaex** [comment]
You mean this?

![](https://jaex.getsharex.com/2022/05/ShareX_CLpIE2Cm32.png)

**Comment by BloodRaven0** [comment]
Yes, my apologies for not looking over the settings before opening this issue, and thanks for the answer! ❤️
